resources/views/addons/index.blade.php

```diff
@@ -2,6 +2,7 @@
 
 @extends('statamic::layout')
 @section('title', __('Addons'))
+@section('wrapper_class', 'max-w-full')
```

Comment:
I think there should still be a max width. Wider than it was, but not full. It's ok on the listing but on the show page it's awkward.

Reply:
Oh, good catch! My browser was only 1/2 of my screen but looking at it now, it's silly. 🙈

Any better now?

Reply:
Yup